.ruby-version

```diff
@@ -1 +1 @@
-3.1.6
+3.3.8
```

Comment:
Best practice is for this to stay in step with the minimum supported version of Ruby. It helps to make sure we aren't accidentally relying on any syntax changes introduced in later versions. That's also covered by CI in this repo, but it's a faster feedback loop here.